bundlex.exs

```diff
@@ -12,19 +25,25 @@ defmodule Membrane.H264.FFmpeg.BundlexProject do
       parser: [
         interface: :nif,
         sources: ["parser.c"],
-        pkg_configs: ["libavcodec", "libavutil"],
+        os_deps: [
+          {get_ffmpeg(), ["libavcodec", "libswresample", "libavutil"]}
+        ],
```

Comment:
Instead of having `get_ffmpeg/0` function, create a function like 
```elixir
defp ffmpeg_os_deps(), do: {get_ffmpeg(), ["libavcodec", "libswresample", "libavutil"]}
```
to avoid repeating yourself. 

If you decide to still have a function with body of `get_ffmpeg/0`, I would suggest to rename it to something like `get_ffmpeg_url/0`